python/dataproc_templates/pubsublite/pubsublite_to_bigquery.py

```diff
@@ -154,7 +162,7 @@ def run(self, spark: SparkSession, args: Dict[str, Any]) -> None:
         # Read
         input_data=(spark.readStream \
             .format(constants.FORMAT_PUBSUBLITE) \
-            .option(f"{constants.FORMAT_PUBSUBLITE}.subscription",f"projects/617357862702/locations/us-west1/subscriptions/{input_subscription}",) \
+            .option(f"{constants.FORMAT_PUBSUBLITE}.subscription",f"projects/{input_project_id}/locations/{region}/subscriptions/{input_subscription}",) \
```

Comment:
Rather than passing multiple parameters for REGION we can simply fetch it via os.getenv

"projects/{input_project_id}/locations/{os.getenv('REGION')}/subscriptions/{input_subscription}"

Reply:
Used os.environ.get instead of passing additional parameter for REGION. Changes are [here](https://github.com/GoogleCloudPlatform/dataproc-templates/pull/658/commits/e0ebb809e805d4ef75bbf9aa50c8454bb1d99ffd)

Reply:
@arindam-google Let us take the whole subscription string from templateProperty

Reply:
Done -> [commit](https://github.com/GoogleCloudPlatform/dataproc-templates/pull/658/commits/7ee38cf6e35e0bd285f547a4c02142684e698386)